pulp_python/app/models.py

```diff
@@ -110,4 +112,4 @@ class PythonImporter(Importer):
     """
 
     TYPE = 'python'
-    projects = models.TextField(null=True)
+    projects = models.TextField(blank=True)
```

Comment:
Can this be blank? What circumstance does that make sense?

Reply:
You're probably right on that one, I suppose it doesn't make a ton of sense.  Will change.